src/renderer/components/main/mainbar/statusbar.tsx

```diff
@@ -0,0 +1,33 @@
+
+/* IMPORT */
+
+import * as React from 'react';
+import {connect} from 'overstated';
+import Main from '@renderer/containers/main';
+import Position from './statusbar_position';
+import WordCount from './statusbar_word_count';
+
+/* STATUSBAR */
+
+const Statusbar = ({ isStatusbar }) => {
+  if (!isStatusbar) {
+    return null;
+  }
+
+  return <div id="mainbar-statusbar" className="layout-header centerer">
+    <div className="multiple grow">
```

Comment:
Maybe `multiple fluid` is more appropriate here.

Reply:
I tried it with fluid but I don't think that would work well for something like a statusbar. It causes all of the items to expand, whereas most of the components I'd imagine going in here (such as the included word count and position components) look better when taking up as little space as possible. Otherwise, there's just a ton of whitespace around the digits which will likely never be filled, but setting a max-width wouldn't seem appropriate either as you never know what kind of files a user may be working with.